### PR TITLE
Add grot guide docs

### PR DESCRIPTION
**Pull request by grafsean**
Adds grot guide shortcode docs, highlights shortcode params to override default copy.

### CODE REVIEW

File: docs/sources/write/shortcodes/index.md
Comment (jdbaldry) [inline docs/sources/write/shortcodes/index.md:336]: Can we move this to be alphabetical in the page table of contents.

File: docs/sources/write/shortcodes/index.md
Comment (jdbaldry) [inline docs/sources/write/shortcodes/index.md:338]:
```suggestion
The `guide` shortcode embeds a Grot guide into the documentation that people can follow to get help and guidance on topics and direction to documentation resources.
```

File: docs/sources/write/shortcodes/index.md
Comment (jdbaldry) [inline docs/sources/write/shortcodes/index.md:341]: https://grafana.com/docs/writers-toolkit/write/style-guide/write-for-developers/#placeholder-variables

```suggestion
{{</* guide name="<NAME>" title="<TITLE>" text="<TEXT>" button="<BUTTON>" */>}}
```

File: docs/sources/write/shortcodes/index.md
Comment (jdbaldry) [inline docs/sources/write/shortcodes/index.md:344]:
```suggestion
The website looks up guide Markdown files from the website repository in the `content/guides` directory.
The shortcode accepts the following parameters:
```

File: docs/sources/write/shortcodes/index.md
Comment (jdbaldry) [inline docs/sources/write/shortcodes/index.md:353]:
```suggestion
If you don't set custom title, text, and button values, the guide introduction screen uses the following default text:
```

File: docs/sources/write/shortcodes/index.md
Comment (jdbaldry) [inline docs/sources/write/shortcodes/index.md:355]: For accessibility, should we replace the image with the actual text for each parameter?

File: docs/sources/write/shortcodes/index.md
Comment (jdbaldry) [inline docs/sources/write/shortcodes/index.md:450]: What do you think about creating a new Grot guides page? The shortcodes reference page is already quite long and this section pretty large.

**Review by jdbaldry — COMMENTED** [review]
Thank you so much for adding this!

If we do split this into a separate page, I have some conceptual documentation I can contribute about how we track Grot guide usage in https://grafanawebsite.grafana.net/d/adtwukqld20hsb/grot-guides?var-session=.%2B&var-env=master&var-guides=$__all&from=now-1h&to=now&timezone=browser&var-query0=1d&var-pages=$__all&var-range=30d&_dash.hideTimePicker

File: docs/sources/write/shortcodes/index.md
Comment (grafsean) [reply docs/sources/write/shortcodes/index.md:355]: It's also to give a visual representation of the grot guide.

File: docs/sources/write/shortcodes/index.md
Comment (grafsean) [reply docs/sources/write/shortcodes/index.md:451]: We can do that, would you like it under `write/grot-guide.md`?

File: docs/sources/write/shortcodes/index.md
Comment (grafsean) [reply docs/sources/write/shortcodes/index.md:336]: Sure, if we don't move it to its own page I'll reorder.

File: docs/sources/write/shortcodes/index.md
Comment (jdbaldry) [reply docs/sources/write/shortcodes/index.md:451]: That makes sense to me :)

**Comment by grafsean** [comment]
@jdbaldry how does that look now?

File: docs/sources/write/grot-guides/index.md
Comment (jdbaldry) [inline docs/sources/write/grot-guides/index.md:63]: I think if this is an example, rather than a copyable template we should say that.
Perhaps something like:

````suggestion
The header structure defines the guide's image and image spacing.
The following example use the image at the URL `/media/guides/grafana-guides-whichgrafana-header.svg`, specifying the alternative text `Grot metrics wizard`, and setting a width of `221` px and height of `131` px.:

```yaml
header:
  image:
    src: /media/guides/grafana-guides-whichgrafana-header.svg
    alt: Grot metrics wizard
    width: 221
    height: 131
```
````

File: docs/sources/write/grot-guides/index.md
Comment (jdbaldry) [inline docs/sources/write/grot-guides/index.md:122]: https://grafana.com/docs/writers-toolkit/write/style-guide/write-for-developers/#wherever-possible-ensure-the-partial-example-is-copy-pasteable:~:text=Don%E2%80%99t%20use%20an%20ellipsis%20(%E2%80%A6)%20or%20three%20periods%20(...)%20to%20omit%20information.

```suggestion
```

File: docs/sources/write/grot-guides/index.md
Comment (jdbaldry) [inline docs/sources/write/grot-guides/index.md:103]: https://grafana.com/docs/writers-toolkit/write/style-guide/write-for-developers/#wherever-possible-ensure-the-partial-example-is-copy-pasteable:~:text=Don%E2%80%99t%20use%20an%20ellipsis%20(%E2%80%A6)%20or%20three%20periods%20(...)%20to%20omit%20information.

```suggestion
```

File: docs/sources/write/grot-guides/index.md
Comment (grafsean) [reply docs/sources/write/grot-guides/index.md:32]: I think we can link to it from the shortcode page with a summary.

Comment (grafsean) [reply docs/sources/write/grot-guides/index.md:32]: What do you think?

Comment (jdbaldry) [reply docs/sources/write/grot-guides/index.md:32]: Sounds great, do you want me to commit that section with the link?

**Review by jdbaldry — APPROVED** [review]
Beautiful, thanks for sticking with me on this one.

It may be nice to get a technical writer less familiar with Grot guides to take a look, either before we merge this or after, to make sure that it helps the target audience.

One final ask, could you add a row to https://grafana.com/docs/writers-toolkit/whats-new/ (https://github.com/grafana/writers-toolkit/blob/main/docs/sources/whats-new.md?plain=1#L23-L25). For a bigger reference, you can refer to https://developers.google.com/style/whats-new

File: docs/sources/write/grot-guides/index.md
Comment (grafsean) [reply docs/sources/write/grot-guides/index.md:32]: I can do it, you've already helped so much!

File: docs/sources/whats-new.md
Comment (jdbaldry) [inline docs/sources/whats-new.md:25]: ❤️

**Pull request merged**
